data_diff/dbt.py

```diff
@@ -18,17 +18,6 @@
 logger = getLogger(__name__)
 
 
-def import_dbt():
```

Comment:
I thought the deletion of this was in master but apparently not. Not needed -- duplicate code